exercises/secret-handshake/src/main/kotlin/HandshakeCalculator.kt

```diff
@@ -0,0 +1,5 @@
+object HandshakeCalculator {
+    fun calculateHandshake(number: Int): List<Signal> {
+        return emptyList()
```

Comment:
Instead, I suggest throwing a [not implemented error](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-not-implemented-error/index.html).

Reply:
Have sense. There is also a specially dedicated [TODO](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-t-o-d-o.html) shortcut. Does not it fir more?